packages/sonataflow-deployment-webapp/webpack.config.js

```diff
@@ -48,4 +56,17 @@ module.exports = async (env) =>
       compress: true,
       port: buildEnv.sonataFlowDeploymentWebapp.dev.port,
     },
+    resolve: {
+      fallback: {
+        https: require.resolve("https-browserify"),
+        path: require.resolve("path-browserify"),
+        http: require.resolve("stream-http"),
+        os: require.resolve("os-browserify/browser"),
+        fs: false,
+        child_process: false,
+        net: false,
+        buffer: require.resolve("buffer/"),
```

Comment:
This is related to the solution below because "swagger-parser" gives a lot of errors and warnings in a webpack environment.  
https://github.com/APIDevTools/swagger-parser/issues/137

But because we have 
`ignoreWarnings: [/Failed to parse source map/],`

the only required fallback is:
`http: require.resolve("stream-http"),`

I will remove the other fallbacks.